packages/analytics/analytics-metric-provider/src/components/MetricsTestHarness.vue

```diff
@@ -67,21 +66,21 @@ const props = withDefaults(defineProps<{
 })
 
 // Query stats for an entire org, no grouping or filtering.
-const globalProviderProps = {
+const globalProviderProps = computed(() => ({
```

Comment:
question: where are these used? couldn't you just use the `props` themselves since these aren't mutated?

Reply:
This is the `MetricsTestHarness`; it's used to render various metric card scenarios during component testing.  This diff is actually fixing a bug in the tests where the test harness was not correctly reacting to prop changes.  I could use the props directly in the specific case of `globalProviderProps`, but I used a `computed` to make it consistent with the other prop cases.